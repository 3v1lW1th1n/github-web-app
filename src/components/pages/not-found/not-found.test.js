import 'jest-styled-components'

import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './'

describe('<NotFound />', () => {
  it('should match default snapshot', () => {
    const component = shallow(<NotFound />)
    expect(component).toMatchSnapshot()
  })

  it('should have the right displayName', () => {
    expect(NotFound.displayName).toEqual('NotFoundPage')
  })
})
