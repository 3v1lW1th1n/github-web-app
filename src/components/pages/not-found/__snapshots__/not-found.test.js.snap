// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should match default snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "NotFoundPage-sc-18uc5oq-0",
        "isStatic": true,
        "rules": Array [
          ".not-found__title{font-family:var(--site-font);font-size:32px;text-align:center;margin:32px 0;}.not-found__content{width:100%;max-width:600px;margin:32px auto;}",
        ],
      },
      "displayName": "NotFoundPage",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "NotFoundPage-sc-18uc5oq-0",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
